html/browsers/windows/browsing-context.html

```diff
@@ -39,10 +41,11 @@
       assert_false(doc.documentElement.childNodes[1].hasChildNodes(), "The BODY element should not have children.");
     }, "Check that new document nodes extant, empty");
 
-    test(function () {
-      assert_equals(doc.referrer, document.URL, "The document's referrer should be its creator document's address.");
+    test(function() {
+      assert_equals(doc.referrer, document.location.origin + '/', "The document's referrer should be its creator document's address's origin.");
```

Comment:
Nit: no need for "address" here.

Reply:
Thanks for the review!

> Nit: no need for "address" here.

Done; thanks for the suggestion

> I think this looks reasonable though I didn't look at all the things. Is this default referrer policy also reflected in fetch requests that end up in a service worker and is that tested?

Yes, this is covered by tests changed in the linked CL, like fetch-event-referrer-policy.https.html.

Reply:
Thanks, any feedback on the remainder of the comment?

Reply:
@maudnals Hey Maud, do you know offhand if Firefox and WebKit have tracking bugs specifically for changing the default referrer policy? Thanks!

Reply:
(There's also the weirdness with the non-JSON-"JSON".)

Reply:
re JSON, I filed issue 26528 for you

Reply:
As mentioned over [here](https://github.com/w3c/webappsec-referrer-policy/pull/142#issuecomment-698539632), Firefox does. I think Safari doesn't because they've already rolled out a superset of this change (discussion on that thread and https://github.com/privacycg/proposals/issues/13). Ultimately this change allows everyone to align more to Safari, and bring Safari's behavior closer to the spec. We'll consider the additional referrer trimming that they do in the PrivacyCG is my understanding

Reply:
I thought Safari didn't do a superset after all as they differed for subresources and navigations?

Reply:
I filed https://bugs.webkit.org/show_bug.cgi?id=218909 against WK. I think it's fair to describe their behavior as "roughly a superset of `strict-origin-when-cross-origin`, with a few rough edges": for instance, there are existing longstanding bugs where various HTML elements' `referrerpolicy` attributes no-op.

Reply:
Yeah I think it is slightly unclear how safari behaves in all cases, given the tracking bug above and at least discrepancies like https://bugs.webkit.org/show_bug.cgi?id=217758. I don't think this change is contentious, but I think the outcome of the referrer trimming discussion is where there are the more divergent views